h2o-py/h2o/model/extensions/std_coef.py

```diff
@@ -4,12 +4,14 @@
 
 class StandardCoef:
 
-    def _std_coef_plot(self, num_of_features=None, server=False):
+    def _std_coef_plot(self, num_of_features=None, server=False, save_plot_path:str=None, **kwargs):
```

Comment:
is this python 2 compatible? I wouldn't think so

Reply:
type annotations: definitely not

Reply:
fixed 
dbd62d0